Control coverage/Feature/VirtualNetwork.md

```diff
@@ -5,4 +5,7 @@
+| <b>ControlId:</b><br>Azure_ERvNet_NetSec_Dont_Enable_IPForwarding_for_NICs<br><b>DisplayName:</b><br>Set 'EnableIPForwarding' flag to false for NICs in the ExpressRoute-connected vNet. <br><b>Description: </b><br> Set 'EnableIPForwarding' flag to false for NICs in the ExpressRoute-connected vNet.| <b>ARM API to list Network Interfaces at <br>subscription level:</b><br>/subscriptions/{subscriptionId}/providers<br>/Microsoft.Network/networkInterfaces<br>?api-version=2019-04-01<br><b>Properties:</b><br>properties.ipConfigurations[\*].properties.subnet.id,<br> properties.enableIPForwarding <br> <br> <b> ARM API to list Virtual Network Gateways at subscription level: </b> <br> /subscriptions/{subscriptionId}/providers/Microsoft.Network/virtualNetworkGateways?<br>api-version=2019-04-01 <br><b>Properties:</b><br> properties.gatewayType | <b>Passed: </b><br>No NICs found on the ERvNet. <br>Or, no NICs found with EnableIPForwarding turned on the ERvNet.<br><b>Failed: </b><br>IP Forwarding is enabled for one or more NIC(s) in ERvNet.<br><b>NotApplicable: </b><br> Current VNet resource object is not connected to ExpressRoute gateway. |
+| <b>ControlId:</b><br>Azure_ERvNet_NetSec_Dont_Use_Multi_NIC_VMs<br><b>DisplayName:</b><br>There must not be multiple NICs on ExpressRoute-connected VMs. <br><b>Description: </b><br> There must not be multiple NICs on ExpressRoute-connected VMs.| <b>ARM API to list Network Interfaces at <br>subscription level:</b><br>/subscriptions/{subscriptionId}/providers<br>/Microsoft.Network/networkInterfaces<br>?api-version=2019-04-01<br><b>Properties:</b><br>properties.ipConfigurations[\*].properties.subnet.id,<br> properties.virtualMachine.id <br> <br> <b> ARM API to list Virtual Network Gateways at subscription level: </b> <br> /subscriptions/{subscriptionId}/providers/Microsoft.Network/virtualNetworkGateways?<br>api-version=2019-04-01 <br><b>Properties:</b><br> properties.gatewayType | <b>Passed: </b><br>No NICs found on the ERvNet. <br>Or, no VMs attcahed to ERvNet have mulitple NICs.<br><b>Failed: </b><br>One or more VMs in the ERvNet are connected to multiple NICs.<br><b>NotApplicable: </b><br> Current VNet resource object is not connected to ExpressRoute gateway. |
```

Comment:
Typo - `attcahed` to be replaced with `attached`.
Typo - `mulitple` to be replaced with `multiple`.

Reply:
Fixed